addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -101,11 +101,15 @@ def _inverse_l10n_latam_document_number(self):
         for rec in self.filtered(lambda x: x.l10n_latam_document_type_id):
             if not rec.l10n_latam_document_number:
                 rec.name = '/'
-            else:
+                continue
+
```

Comment:
NBD: I would just personally avoid adding this diff, even if in terms of lines changed it's pretty much equivalent, just so that the original structure remains visible for anyone interested in looking at the file's history

Reply:
I made just to avoid unnecessary computation by calling `_format_document_number`. But if you want I can implement it like:

```diff
     def _inverse_l10n_latam_document_number(self):
         for rec in self.filtered(lambda x: x.l10n_latam_document_type_id):
             if not rec.l10n_latam_document_number:
                 rec.name = '/'
             else:
                 l10n_latam_document_number = rec.l10n_latam_document_type_id._format_document_number(rec.l10n_latam_document_number)
-                if rec.l10n_latam_document_number != l10n_latam_document_number:
+                if not rec._skip_format_document_number() and rec.l10n_latam_document_number != l10n_latam_document_number:
                     rec.l10n_latam_document_number = l10n_latam_document_number
                 rec.name = "%s %s" % (rec.l10n_latam_document_type_id.doc_code_prefix, l10n_latam_document_number)

```